parsers/SG.py

```diff
@@ -0,0 +1,229 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from collections import defaultdict
+
+import arrow
+import requests
+
+
+# REMAINING TODO: integrate in other files
```

Comment:
Nope, sorry, please remove this line. I meant the README and the config, but forgot to take it out once done...